README.md

```diff
@@ -0,0 +1,62 @@
+# Picarro Data Upload and Download
+This github repository contains all scripts for uploading and downloading data from Google Cloud Storage and Firestore
+
+* The firestore database consists of *hourly aggregate* data for the Picarro machine,combined into `Year/Month` collections
+* Google Cloud Storage holds the raw data with no aggregation
+
+## Setting Up Your Machine
+To run these scripts, you'll need the following:
+* Python version 3.5 or greater. This may be pre-installed on your system; to check use `python3 --version`
+
+Once you've checked that you have the proper version of python installed, create and activate a virtual environment:
+
+Mac:
+```
+python3 -m venv venv
+source venv/bin/activate
+```
+Windows:
+```
+python3 -m venv .venv
+source .venv\Scripts\activate.bat
+```
+After activating your virtual environment, you'll need to install the git repository:
+```pip install https://github.com/brown-ccv/picarro-data.git```
+
+
+## Logging in to Google Cloud
+In order to upload and download files, you'll need to log in to Google Cloud. First, you'll need to be added to the project on firebase. Grace Berg, Ruby Ho, and Sam Bessey can add you.
+
+There are two options:
+1. Navigate to the Firebase console to generate a private key (Settings > Service Accounts).
+2. Download the `serviceAccount.json` file to a convenient location.
+3. Run `export GOOGLE_APPLICATION_CREDENTIALS="KEY_PATH"` where `KEY_PATH` is the location you stored your file. Example: `export GOOGLE_APPLICATION_CREDENTIALS="~/Documents/picarro/serviceAccount.json"`
+
+Alternately, run `gcloud auth application-default login --impersonate-service-account firebase-adminsdk-u3hum@hastings-picarro.iam.gserviceaccount.com` and follow the steps to log in.
```

Comment:
Does this service account have limited read-only permissions, different from the one that you use to upload the datat?